### PR TITLE
Updated README.md with logo

### PR DESCRIPTION
**Pull request by 12-malak**
Added the logo in the README.md of this repo for issue #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Hacktober Fest 2022 Participants
 
+![OSSPH_Hacktober](https://user-images.githubusercontent.com/73274076/194005264-5e2429d2-c6de-41fc-b5e3-671afc16f60a.png)
```
Comment (jofftiquez) [inline README.md]: The image must be uploaded in the root directory and the just a link. You can use the `img` html tag display the logo

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (12-malak) [reply README.md]: Got it! Working on it.

**Review by 12-malak — COMMENTED** [review]
@jofftiquez Added img html link for logo in README.md, as requested issue #2

**Comment by 12-malak** [comment]
@jofftiquez  Done requested changes. Please review and merge. Also, please feel free to add the hacktoberfest-accepted label once approved.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Hacktober Fest 2022 Participants
 
+<img src="https://github.com/12-malak/hacktober-fest-2022-participants/blob/main/Logo.png" alt="hi" class="inline"/>
```
Comment (jofftiquez) [inline README.md]: The image should be in the root foolder of this repo. Not a link from a different repository. 

Also, kindly post the logo in the PR as well thanks.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

**Pull request merged**